plugin/plugins/load_tester/__init__.py

```
@@ -0,0 +1,1564 @@
+        max_count: int = 50,
+        plugin_id: str = "*",
+        timeout: float = 0.5,
+        **_: Any,
+    ):
+        root_cfg = self._get_load_test_section(None)
+        sec_cfg = self._get_load_test_section("bus_messages_get")
+
+        try:
+            self.ctx.close()
+        except Exception:
+            pass
```

Ruff (0.14.10)

`try`-`except`-`pass` detected, consider logging the exception

(S110)